website/src/components/Chat/ChatConversation.tsx

```diff
@@ -116,11 +121,96 @@ export const ChatConversation = memo(function ChatConversation({ chatId, getConf
     },
     [getConfigValues, setIsSending, setShowEncourageMessage, toast]
   );
+  const createAssistantDrafts = useCallback(
```

Comment:
I wonder if this will be a problem when plugins are enabled... With sufficient retries/depth plugins take up a lot of model time/compute already, several drafts might be a struggle. Maybe we could reduce plugin max depth for drafts?

Reply:
I might just disable drafts when plugins are enabled until the disabling under load feature is added as to not overload the inference server. Reducing max iterations is also a possibility.

Reply:
> I might just disable drafts when plugins are enabled until the disabling under load feature is added as to not overload the inference server. Reducing max iterations is also a possibility.

Yes, not only overload but also just taking a lot of time for the user to get the drafts... Maybe include a setting to toggle enabling this for plugins, and have it disabled by default

Reply:
I've added an ENABLE_DRAFTS_FOR_PLUGINS environment variable which is defaulted to false